### ISSUE TITLE
AttributeError: has no attribute '_errors'

### ISSUE THREAD

**Issue by clayg** [iamteem/redisco#4]
The errors property on models.base should check for _errors first, and if not exists call .is_valid for you

I can:
x = MyModel.objects.create()
x.errors # == []

but not:
x = MyModel()
x.errors # raises AttributeError!

**Comment by clayg** [comment]
failing test:
http://github.com/clayg/redisco/commit/4adf5eb8aaf82759227db2274727f1d080a12b17

**Comment by clayg** [comment]
I don't think that accessing x.errors before calling x.is_valid should raise an attribute error for missing attribute "_errors":
http://github.com/clayg/redisco/compare/master...issue4
pull request sent.

But I'm not sure how effectively errors can be cached?  Maybe the property should always run the full validation suite before returning to prevent divergence?  Meh...

**Comment by rhgrant10** [comment]
For what it's worth, I experienced the same error after creating a `ModelForm` and forgetting to call `super` in `__init__`, so I'm pretty glad it didn't attempt to fix the problem by calling `is_valid`. It would have been quite wrong and likely led to an even more obscure message.
